ports/raspberrypi/common-hal/microcontroller/Pin.c

```diff
@@ -28,18 +28,34 @@
 
 #include "common-hal/microcontroller/__init__.h"
 #include "shared-bindings/microcontroller/Pin.h"
+#include "bindings/cyw43/__init__.h"
 
 #include "src/rp2_common/hardware_gpio/include/hardware/gpio.h"
 
+#if CIRCUITPY_CYW43
+#include "pico/cyw43_arch.h"
+#endif
+
 STATIC uint32_t never_reset_pins;
 
+bool cyw_ever_init;
+static uint32_t cyw_pin_claimed;
+
```

Comment:
Guard with `CIRCUITPY_CYW43` so it's completely not there if compiling for regular Pi Pico?